handler/status.go

```diff
@@ -73,6 +75,20 @@ func Status(
 				"base64": func(s string) string {
 					return base64.RawURLEncoding.EncodeToString([]byte(s))
 				},
+				"formatHistogram": func(m *dto.Histogram) string {
+					h, fh := histogram.NewModelHistogram(m)
+					if h == nil {
+						return fmt.Sprintf("{ count:%v sum:%v %s}",
+							m.GetSampleCountFloat(),
+							m.GetSampleSum(),
+							histogram.BucketsAsString[float64](histogram.GetAPIFloatBuckets(fh)))
+					} else {
```

Comment:
This `else` isn't really needed, is it? (You `return` in the `if` block above.)

Reply:
That's the else branch if we don't have a histogram but a float histogram.

Reply:
I mean, don't remove the whole block, just remove the `} else {` and the closing  `}` below and outdent the content of the block.